deploy/tasks/users.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
perhaps we can keep user configuration separate from this (users can be considered as application data, not infrastructure configuration)